Resources/text/fr/lang_fr.txt

```diff
@@ -129,42 +129,42 @@ GB_nOT = Dernier Dresseur connu
 Label_PrevOT = DO :
 GB_ExtraBytes = Octets supplémentaires
 GB_Markings = Marquages
-BTN_Ribbons = Rubans / Médaill.
+BTN_Ribbons = Rubans / Médailles
 BTN_History = Souvenirs/Récré
-Label_EncryptionConstant = Valeur de cryptage :
-BTN_RerollEC = Edit.
+Label_EncryptionConstant = Constante de cryptage :
+BTN_RerollEC = Édition
 -----------------Save Data Interface--------------------
 Tab_Box = Boîtes PC
-Tab_PartyBattle = Partie
+Tab_PartyBattle = Pokémons en votre possession
```

Comment:
Should double check that the lengthened text doesn't overflow/clip out of their respective element. Menu Strips are pretty flexible; however Buttons, Tabs, and Labels aren't.

Be sure that all the supplied text is what appears in the program! Feel free to abbreviate / summarize instead of being precise.